jmix-dynattr/dynattr-flowui/src/main/java/io/jmix/dynattrflowui/view/categoryattr/CategoryAttributesDetailView.java

```diff
@@ -302,7 +308,10 @@ protected List<String> getEnumValues() {
 
     @Subscribe
     protected void onAfterShow(BeforeShowEvent event) {
-        initDefaultEnumField();
+        if (getEditedEntity().getDataType() != null &&
```

Comment:
as we discussed, yes this is right code, but we need to relocate this part into the method that will be named "initDefaultEnumFieldIfEnum"